jsr.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@eslint/markdown",
+  "version": "6.0.0",
+  "exports": "./dist/esm/index.js",
+  "publish": {
+    "include": [
+      "dist/esm/index.js",
+      "dist/esm/index.d.ts",
+      "dist/esm/types.ts",
```

```suggestion
      "dist/esm/types.ts",
      "dist/esm/types.d.ts",
```

Should maybe `dist/esm/types.d.ts` also be published?

---

I'm not sure, but doesn't hurt to include it.